File: JavaScript/Ex.8.js
let matriz = [[0,0], [0,0], [0,0]]

for(let l = 0; l < 3; l++){
  for(let c = 0; c < 2; c++){
    matriz[l][c] = prompt("Insira um número: ")
  }
}

for(let l = 0; l < 3; l++){
  for(let c = 0; c < 2; c++){
    if(matriz[l][c] % 3 == 0){
      console.log("[" + matriz[l][c] + "]")
    }
  }
}

File: Python/Ex.8.py
matriz = [[None, None], [None, None], [None, None]]

for l in range(0, 3):
  for c in range(0, 2):
    matriz[l][c] = int(input("Insira um número: "))

for l in range(0,3):
  for c in range(0,2):
    if matriz[l][c] % 3 == 0:
      print(f"[{matriz[l][c]}]")